packages/windows/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.16.1"
```

```suggestion
- version: "1.17.0"
```

because this is an enhancement.